src/components/Header/Notifications/Modal/Modal.js

```diff
@@ -45,63 +51,80 @@ class Modal extends Component {
                 <h2>Notification</h2>
               </div>
               <br />
-              <div />
 
-              {notifications.length === 0 ? (
-                <div>
-                  <h3 className="center">No new notifications</h3>
-                  <div className="divider" />
-                </div>
-              ) : (
-                notifications.map((val, key) => (
-                  <div key={key} className={val.status === 'unseen' ? 'row light-red' : 'row'}>
-                    <div role="button" onClick={() => this.updateStatus(val)}>
-                      <div key={key} className="wrap-notification">
-                        <div className="notification-tick">
-                          {val.status === 'unseen' ? (
-                            <FontAwesomeIcon
-                              icon={faCheck}
-                              size="1x"
-                              className="text-light-grie small-padding left"
-                            />
-                          ) : (
+              {notifications !== undefined
+                && (notifications.length === 0 ? (
+                  ' '
+                ) : (
+                  <div>
+                    <span>Mark all notification as seen:</span>
+                    <span>
+                      <button
+                        className="border b-light medium-margin radius-2"
+                        onClick={this.markAsSeen}
+                      >
+                        <FontAwesomeIcon icon={faCheckDouble} size="2x" className="text-success" />
+                      </button>
+                    </span>
+                  </div>
+                ))}
+              <div />
+              {notifications !== undefined
+                && (notifications.length === 0 ? (
+                  <div>
+                    <h3 className="center">No new notifications</h3>
```

Comment:
why are we passing new notification object

Reply:
also we can abstract this

Reply:
resolved

Reply:
how has this been resolved?

Reply:
I was using if statement twice yet I can combine them in one statement.

Reply:
why are we passing a new notification object?

Reply:
If you mean the "new notifications" on line 75, that is not an object. It is a message that is displayed when there is no notification